### PR TITLE
Bugfix/mob 3057 company name config in intent

### PR DESCRIPTION
**Pull request by Pelkar**
**Jira issue:**
https://glia.atlassian.net/browse/MOB-3057

**What was solved?**

Lost "Company name" configurability through activity intent was reintroduced

**Release notes:**

 - [ ] Feature
 - [ ] Ignore
 - [X] Release notes (Is it clear from the description here?)
 - [ ] Migration guide (If changes are needed for integrator already using the SDK - what needs to be communicated? Add underneath please)

**Additional info:**

 - [ ] Tests updated? Added? Unit, acceptance, snapshots?
 - [ ] Logging for future troubleshooting of client issues added?

**Screenshots:**

### CODE REVIEW

**Review by DavDo — APPROVED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/core/configuration/GliaSdkConfigurationManager.java [hunk 1]
```diff
@@ -30,16 +31,26 @@ public void setUseOverlay(boolean enabled) {
...
+        this.legacyCompanyName = companyName;
+    }
+
     /** @noinspection StatementWithEmptyBody*/
     public String getCompanyName() {
         String remoteCompanyName = fetchRemoteCompanyName();
         boolean isCompanyNameSetFromGliaHub = remoteCompanyName != null && !remoteCompanyName.trim().isEmpty();
 
+        Logger.d("CONFIGURATION", "companyName =" + companyName);
+        Logger.d("CONFIGURATION", "legacyCompanyName =" + legacyCompanyName);
```
Comment (andrews-moc) [inline widgetssdk/src/main/java/com/glia/widgets/core/configuration/GliaSdkConfigurationManager.java:44]: Can these logs be removed or you think we need them?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <string name="app_name">Glia SDK Widget Example App</string>
 
     <string name="settings_value_default">Default</string>
-    <string name="settings_value_default_company_name">Glia company</string>
+    <string name="settings_value_default_company_name">Init company name</string>
```
Comment (andrews-moc) [inline app/src/main/res/values/strings.xml:7]: Shall it also be changed on iOS?

**Review by AndriiHorishniiMOC — APPROVED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/core/configuration/GliaSdkConfigurationManager.java (hunk 1 above)
Comment (Pelkar) [reply widgetssdk/src/main/java/com/glia/widgets/core/configuration/GliaSdkConfigurationManager.java]: Removed testing logs

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (Pelkar) [reply app/src/main/res/values/strings.xml:7]: We can suggest, but the point of this is to implicitly shoiw where the value is coming from

**Pull request merged**